app/rdd2/src/casadi/rdd2.py

```diff
@@ -457,6 +467,125 @@ def derive_strapdown_ins_propagation():
     }
     return eqs
 
+def derive_attitude_estimator():
```

Comment:
It is good to state your original source for this approach, paper/ source code and state what you did and any modifications you made to their approach.